frontend/src/components/ReactivateVaultDialog.vue

```diff
@@ -82,7 +82,7 @@ async function reactivateVault() {
   const dto =  { ...props.vault };
   dto.archived = false;
   try {
-    const updatedVault = await backend.vaults.createOrUpdateVault(dto);
+    const updatedVault = await backend.vaults.createOrUpdateVault(dto, true, true);
```

Comment:
refine aws/minio arg? dto does not have corresponding information, need to extract metadata.